### PR TITLE
Add tooltip children

### PR DESCRIPTION
**Pull request by lesbaa**
- add ability to add react elements as children to tooltips
- update docz

### CODE REVIEW

File: src/components/Tooltip/DemoElement.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/* eslint-disable */
+import React, { Fragment } from 'react'
+
+const DemoElement = (props) => (
```
Comment (SamDavies) [inline src/components/Tooltip/DemoElement.js]: remove extra brackets on props and function

File: src/components/Tooltip/index.js [hunk 2]
```diff
@@ -60,8 +61,15 @@ export const StyledTooltip = styled.div`
     }
 `
 
-const Tooltip = props => (
-    <TooltipTrigger
+const Tooltip = props => {
+    const tooltipContent = isString(props.tipText)
```
Comment (SamDavies) [inline src/components/Tooltip/index.js]: lets get rid of this strange code, we can pass `<p>` tags in the children if needed

Comment (lesbaa) [reply src/components/Tooltip/index.js]: @SamDavies The problem is that the current API uses children for the content of the trigger and `tipText` for the content so if we can change this API, it'll need changed across the whole app as the changes will be breaking.  I'll try and figure out a better API and then add a complimentary PR in the app once it's approved.

Comment (lesbaa) [reply src/components/Tooltip/index.js]: maybe something like:
```js
<Tooltip>
  <TooltipContent>
    // ... whatever elements you want in here
  </TooltipContent>
  <TooltipTrigger>
    // ... button or something
  </TooltipTrigger>
</Tooltip>
```

Comment (SamDavies) [reply src/components/Tooltip/index.js]: perhaps:
```
<Tooltip
    renderTooltip={() => <div>...</div>}
>
  <TooltipTrigger>
    // ... button or something
  </TooltipTrigger>
</Tooltip>
```

**Pull request closed without merging**